documentation/asciidoc/computers/raspberry-pi/power-supplies.adoc

```diff
@@ -73,64 +96,60 @@ Exactly how much current (mA) the Raspberry Pi requires depends on what you conn
 | | Avg | 0.22 | 0.28 | 0.33 | 0.16 | 0.78 
 | Stress | Max | 0.35 | 0.82 | 1.34 | 0.35 | 1.25 
 | | Avg | 0.32 | 0.75 | 0.85 | 0.23 | 1.2 
 | Halt current | | | | 0.10 | 0.055 | 0.023 
 |===
 
-NOTE: For these measurements we used a standard Raspberry Pi OS image (current as of 26 Feb 2016, or June 2019 for the Raspberry Pi 4), at room temperature, with the Raspberry Pi connected to a HDMI monitor, USB keyboard, and USB mouse. The Raspberry Pi 3 Model B was connected to a wireless LAN access point, the Raspberry Pi 4 was connected to Ethernet. All these power measurements are approximate and do not take into account power consumption from additional USB devices; power consumption can easily exceed these measurements if multiple additional USB devices or a HAT are connected to the Raspberry Pi.
+NOTE: These measurements used a standard Raspberry Pi OS image (current as of 26 Feb 2016, or June 2019 for the Raspberry Pi 4), at room temperature, with the Raspberry Pi connected to a HDMI monitor, USB keyboard, and USB mouse. The Raspberry Pi 3 Model B was connected to a wireless LAN access point, the Raspberry Pi 4 was connected to Ethernet. All these power measurements are approximate and do not take into account power consumption from additional USB devices; power consumption can easily exceed these measurements if multiple additional USB devices or a HAT are connected to the Raspberry Pi.
 
 [.whitepaper, title="Extra PMIC features on Raspberry Pi 4 and Compute Module 4", subtitle="", link=https://pip.raspberrypi.com/categories/685-whitepapers-app-notes/documents/RP-004340-WP/Extra-PMIC-features-on-Raspberry-Pi-4-and-Compute-Module-4.pdf]
 ****
 A number of different PMIC devices have been used on both Raspberry Pi 4 and CM4. All the PMICs provide extra functionality alongside that of voltage supply. This document describes how to access these features in software.
 ****
 
+==== Decrease Raspberry Pi 5 wattage when turned off
+
+By default, the Raspberry Pi 5 consumes around 1W to 1.4W of power when turned off. This can be decreased by manually editing the EEPROM configuration with `sudo rpi-eeprom-config -e`. Change the settings to the following:
+
+[source]
+----
+BOOT_UART=1
+POWER_OFF_ON_HALT=1
+BOOT_ORDER=0xf416
+----
+
+This should drop the power consumption when powered down to around 0.01W.
+
 === Power supply warnings
 
 On all models of Raspberry Pi since the Raspberry Pi B+ (2014) except the Zero range, there is low-voltage detection circuitry that will detect if the supply voltage drops below 4.63V (±5%). This will result in an entry being added to the kernel log.
 
-If you are seeing warnings, you should improve the power supply and/or cable, as low power can cause problems with corruption of SD cards, or erratic behaviour of the Raspberry Pi itself.
+If you see warnings, switch to a better power supply and cable. Low power can corrupt storage or cause unpredictable behaviour within the Raspberry Pi.
```

Comment:
I realise this is just the rework of the existing text - but I'm sort of uncomfortable with using 'better' and 'low' here - presumably we actually mean 'higher quality' and 'lower quality', and that comes through clearly enough, but it just reads a little too blunt.

Reply:
Agreed. I think this need... softening. @nathan-contino Can you take another pass here?

Reply:
Updated and softened.